iiop-openjdk/src/main/java/org/wildfly/iiop/openjdk/rmi/ValueAnalysis.java

```diff
@@ -173,7 +173,7 @@ protected void doAnalyze() throws RMIIIOPViolationException {
                     && wo.getReturnType() != Void.TYPE) {
                 wo = null; // Wrong return type
             }
-            if (wo != null) { // Right modifiers?
+            if (wo != null && spf != null) { // Right modifiers?
```

Comment:
@tadamski Please review.

I suspect this isn't the right fix here and that the problem is that L177 should be

int mods = wo.getModifiers();

I don't see why the modifiers of the 'serialPersistentFields' field would be relevant at this point.